core/src/test/scala/unit/kafka/message/MessageTest.scala

```diff
@@ -66,26 +65,31 @@ class MessageTest extends JUnitSuite {
       } else {
         TestUtils.checkEquals(ByteBuffer.wrap(v.payload), v.message.payload)
       }
+
       // check timestamp
-      if (v.magicValue > Message.MagicValue_V0)
+      if (v.magicValue > Message.MagicValue_V0) {
         assertEquals("Timestamp should be the same", v.timestamp, v.message.timestamp)
-       else
+      } else {
         assertEquals("Timestamp should be the NoTimestamp", Message.NoTimestamp, v.message.timestamp)
+      }
```

Comment:
Just a habit. Many times I got burned by code like this:
```
if ( p == 1 )
  doWork()
else
  somethingElse()
  thenOtherStuff()
```
If you quickly read code you might not immediately see that `thenOtherStuff()` does not belong to else block. Therefore it is good practice to always use 
```
if ( ... ) { 
... 
} else { 
... 
}
```